suite-native/notifications/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@suite-native/notifications",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
+        "type-check": "tsc --build"
+    },
+    "dependencies": {
+        "@react-navigation/native": "^6.1.3",
+        "@suite-common/wallet-config": "workspace:*",
+        "@suite-common/wallet-core": "workspace:*",
+        "@suite-common/wallet-types": "workspace:*",
+        "@suite-native/atoms": "workspace:*",
+        "@suite-native/formatters": "workspace:*",
+        "@suite-native/navigation": "workspace:*",
+        "@suite-native/transactions": "workspace:*",
+        "@trezor/icons": "workspace:*",
+        "@trezor/styles": "workspace:*",
+        "@trezor/theme": "workspace:*",
+        "react": "18.2.0",
+        "react-native": "0.71.3",
+        "react-native-gesture-handler": "^2.9.0",
+        "react-native-reanimated": "2.14.4",
+        "react-redux": "7.2.2"
```

Comment:
You should use version 8.0.5 or there will be version mismatch throughout the repo

Reply:
fixed 11399f17d